hawkular-accounts-feature-pack-dist/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.hawkular.accounts</groupId>
+    <artifactId>hawkular-accounts-parent</artifactId>
+    <version>2.0.5.Final-SNAPSHOT</version>
+  </parent>
+  <artifactId>hawkular-accounts-feature-pack-dist</artifactId>
+  <name>Hawkular - Accounts - Feature Pack Distribution</name>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.hawkular.accounts</groupId>
+      <artifactId>hawkular-accounts-feature-pack</artifactId>
+      <version>${project.version}</version>
```

Comment:
Why not 2.0.5.Final-SNAPSHOT? AFAIK, the recommended is to use the version string, instead of a property for this situation. The maven release can handle this update.